infinex-omnibus/staging/infinex-omnibus-test.polygon.toml

```diff
@@ -0,0 +1,22 @@
+#  pnpm cannon build infinex-omnibus/staging/infinex-omnibus-test.polygon.toml --chain-id 137 --dry-run
+#  pnpm cannon publish infinex-omnibus-test:latest@omnibus --chain-id 137 --exclude-cloned
+
+name = "infinex-omnibus-test"
+version = "3.0.0"
+preset = "omnibus"
+description = "All Infinex Deployments"
+
+[pull.AccountFactoryV1]
+source = "infinex-account-factory-test:1.0.0@main"
+
+[pull.CoreV1]
+source = "infinex-multichain:1.0.0staging@O2"
+
+[pull.CoreV2]
+source = "infinex-multichain:1.1.0staging@O2"
+
+[pull.CoreV3]
+source = "infinex-core-test:3.0.0@core"
+
+[pull.PatronPurchaseAppV1]
+source = "infinex-patron-test:1.0.0@patron-purchase"
```

Comment:
1.0.1?

Reply:
@lemoninfinex 1.0.0 for staging is right